walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +29,29 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("   Ada   ");
+
+    // Whitespace should be trimmed
```

Comment:
nit: probably don't need this comment since the test method has a descriptive name!